dali/test/operator_arguments.h

```diff
@@ -0,0 +1,21 @@
+#ifndef DALI_OPERATOR_ARGUMENTS_H
+#define DALI_OPERATOR_ARGUMENTS_H
+
+#include <map>
+
+namespace dali {
+
+namespace testing {
+
+
+class OperatorArguments {
+ public:
+  using Arguments = std::map<std::string, double>;
```

Comment:
Something more elaborate here is a must! Any? Variant?

Reply:
Accidentaly I removed TODO here. I've put it back